src/Tests/CaptchaPersistenceTestCase.php

```diff
@@ -203,7 +203,6 @@ public function testPersistenceOnlyOnce() {
     $this->assertPreservedCsid($captcha_sid_initial);
 
     // Start a new form instance/session
-    $this->drupalGet('node');
```

Comment:
The comment says it is testing for captchas on forbidden admin pages. The drupalGet returns a 403 as expected, but I don't get how there is supposed to be a captcha there then.